src/styles/dictionary_modal.scss

```diff
@@ -1,16 +1,22 @@
 .sidenav-concept-btn {
     margin-top: 30%;
  }
- .modal-fade {
-     padding-top: 10%;
-     opacity: 10 !important;
-     background-color: rgba(0, 0, 0, 0.5);
- }
- .modal-content {
-     width: 150%;
-     margin-left: -36%;
-     margin-top: 20%  
- }
+.dictionary-modal-fade {
+    opacity: 10 !important;
+    background-color: rgba(0, 0, 0, 0.5);
+}
+.dictionary-modal-fade form {
+   margin: 0px 3%;
+}
+.dictionary-custom-modal {
+    max-width: 750px;
+    margin-top: 270px;
+}
+.modal-fade {
```

Comment:
Looks like you renamed this class. Is it still necessary?

Reply:
Yes, this class is used in other parts of the codebase.